src/main.ts

```diff
@@ -212,6 +214,62 @@ async function main() {
 		return await handler.fetchBalance(hash, address);
 	});
 
+	// Get generalized staking information.
+	//
+	// Paths:
+	// - (Optional) `number`: Block hash or height at which to query. If not provided, queries
+	//   finalized head.
+	//
+	// Returns:
+	// - `at`: Block number and hash at which the call was made.
+	// - `validatorCount`: Upper bound of validator set size; considered the ideal size.
+	// - `activeEra`: EraIndex of the current era being rewarded.
+	// - `forceEra`: Current status of era forcing.
+	// - `nextEra`: **Upper bound estimate** of the block height at which the next era will start.
+	// - `nextSession`: **Upper bound estimate** of the block height at which the next session will
+	//		start.
+	// - `unappliedSlashes`: Array of upcoming `UnappliedSlash` indexed by era.
+	//  	Each `UnappliedSlash` contains:
+	//		- `validator`: Stash accountId of the offending validator.
+	//		- `own`: The amount the validator will be slashed.
+	//		- `others`: Array of tuples of (accountId, amount) representing all the stashes of other
+	//		slashed stakers an and the amount they will be slashed.
+	//		- `reporters`: Array of accountIds of the reporters of the offence (bounty payout
+	//		recipients.)
+	//		- `payout`: Amount of bounty payout.
+	// - `queuedElected`: Array of accountIds for validators who have been elected. Only available
+	//		when the `ElectionResult` is available.
```

Comment:
This is a bit missleading, `QueuedElected` hold the best phragmen solution yet submitted, but if a better solution is submitted next block then previous solution doesn't hold anymore.